webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/DataStructures.scala

```diff
@@ -616,9 +616,10 @@ final case class BoundPolicyDraft(
                                            )
                                          }
                                          // check if there is mandatory variable with missing/blank values
+                                         // NOTE : it's OK that system variables are not filled here yet
                                          (
-                                           variable.spec.constraint.mayBeEmpty,
-                                           variable.values.exists(v => v == null || v.isEmpty)
+                                           variable.spec.constraint.mayBeEmpty || variable.spec.isSystem,
```

Comment:
skip system variable here, because they can be late bound. They will go in the "true" case, ie like if they are optionnal.